SimGeneral/PreMixingModule/python/mixOne_premix_on_sim_cfi.py

```diff
@@ -337,6 +338,18 @@
     )
 )
 
+phase2_hfnose.toModify(mixData,
```

Comment:
@makortel FYI, we should check if the premixing setup works out of the box for this new detector (an HGCal-like section in front of HF)

Reply:
@kpedro88 From visual inspection I'd expect so (but of course it should be tested). Does this require the v9 geometry of HGCal (for which the premixing still needs to be fixed #23624)?

Reply:
So far the HFNose is only in a Phase2 geometry with HGCal v9.

Reply:
Ok, se we need to fix #23624 first.